Cargo.toml

```diff
@@ -61,8 +61,8 @@ hickory-resolver = "0.24"
 humansize = "2"
 image = { version = "0.25.1", default-features=false, features = ["gif", "jpeg", "ico", "png", "pnm", "webp", "bmp"] }
 iroh_old = { version = "0.4.2", default-features = false, package = "iroh"}
-iroh-net = "0.16.2"
-iroh-gossip = { version = "0.16.2", features = ["net"] }
+iroh-net = { git = "https://github.com/n0-computer/iroh", branch="main" }
```

Comment:
But this way things can break at any moment. Maybe specify the current HEAD commit?